packages/web/app/views/programRegistry/RemoveProgramRegistryFormModal.jsx

```diff
@@ -121,7 +121,7 @@ export const RemoveProgramRegistryFormModal = ({ patientProgramRegistration, onC
             <Value>
               <TranslatedReferenceData
                 fallback={patientProgramRegistration.programRegistry?.name}
```

Comment:
Does this mean the `fallback` doesn't resolve correctly in this context too? Not sure what the fix should be there in that case.

Reply:
I think that resolving to the placeholder is the correct behaviour as it's the values that is `undefined` not the translation. I think in this case it is ok that the placeholder `-` shows untranslated since it's not english text.